Questionable example of named variable `_` in the same scope as a discard

At the very end of this page, we have 

> ... If an identifier named `_` is already in scope, the use of `_` as a standalone discard can result in:

followed by a list of 3 bullets, the final one of which contains the following example:

> ```csharp
> public void DoSomething(int _)
> {
>  var _ = GetValue(); // Error: cannot declare local _ when one is already in scope
> }
> ```

Yes, this does result in error CS0136, but unless I'm missing something, I contend this example has **nothing whatsoever to do with discards, and the comment seems to agree!** The value parameter is named `_`, and is *not* a discard. `var _ = ...` declares `_` to be a local variable (not a discard). And as we *cannot* have two variables with the same name be declared in the same scope, we get CS0136. We get exactly the same error if we change both occurrences of `_` to `x`!

If my analysis is correct, this whole bullet should be removed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7ace3d2c-a7f7-c667-5f50-b8d8f1dfc87a
* Version Independent ID: 6edd2f85-a04b-e9ad-2fc9-fc23dbfc5bba
* Content: [Discards - unassigned discardable variables](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards)
* Content Source: [docs/csharp/fundamentals/functional/discards.md](https://github.com/dotnet/docs/blob/main/docs/csharp/fundamentals/functional/discards.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**


---
[Associated WorkItem - 33090](https://dev.azure.com/msft-skilling/Content/_workitems/edit/33090)

--- a/docs/csharp/fundamentals/functional/discards.md
+++ b/docs/csharp/fundamentals/functional/discards.md
@@ -1,7 +1,7 @@
 ---
 title: Discards - unassigned discardable variables
 description: Describes C#'s support for discards, which are unassigned, discardable variables, and the ways in which discards can be used.
-ms.date: 11/14/2023
+ms.date: 02/19/2025
 f1_keywords:
   - "discard_CSharpKeyword"
 ---
@@ -74,8 +74,6 @@ Without assigning the task to a discard, the following code generates a compiler
    :::code language="csharp" source="snippets/discards/standalone-discard2.cs" ID="VariableIdentifier" :::
 - A compiler error for violating type safety. For example:
    :::code language="csharp" source="snippets/discards/standalone-discard2.cs" ID="VariableTypeInference" :::
-- Compiler error CS0136, "A local or parameter named '\_' cannot be declared in this scope because that name is used in an enclosing local scope to define a local or parameter." For example:
-   :::code language="csharp" source="snippets/discards/standalone-discard2.cs" ID="CannotRedeclare" :::
 
 ## See also
 

--- a/docs/csharp/fundamentals/functional/snippets/discards/standalone-discard2.cs
+++ b/docs/csharp/fundamentals/functional/snippets/discards/standalone-discard2.cs
@@ -32,21 +32,5 @@ public class DiscardOrVariable
     //      error CS0029: Cannot implicitly convert type 'bool' to 'int'
     // </VariableTypeInference>
     
-    // <CannotRedeclare>
-    public void DoSomething(int _)
-   {
-   	var _ = GetValue(); // Error: cannot declare local _ when one is already in scope
-   }
-   // The example displays the following compiler error:
-   // error CS0136:
-   //       A local or parameter named '_' cannot be declared in this scope
-   //       because that name is used in an enclosing local scope
-   //       to define a local or parameter
-   // </CannotRedeclare>
     */
-
-    private int GetValue()
-   {
-      return 3;
-   }
 }

